extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -555,19 +565,19 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
       const contacts: ContactPreview[] = await ContactStore.search(undefined, { substring });
       this.view.errModule.debug(`searchContacts substring: ${substring}`);
       this.view.errModule.debug(`searchContacts db count: ${contacts.length}`);
-      this.renderSearchRes(input, contacts, { substring });
+      await this.renderSearchRes(input, contacts, { substring });
       if (contacts.length >= this.MAX_CONTACTS_LENGTH) {
         this.view.errModule.debug(`searchContacts 2, count: ${contacts.length}`);
         return;
       }
       let foundOnGoogle: EmailProviderContact[] = [];
-      if (this.googleContactsSearchEnabled) {
+      if (await this.googleContactsSearchEnabled || await this.googleContactsSearchEnabled === undefined) {
```

Comment:
please....
`(await this.googleContactsSearchEnable) !== false`

Reply:
Yeah, That looks more clear.